### PR TITLE
Add readthedocs documentation for develop branch

### LINKED ISSUE #49: Create a readthedocs documentation project for the application

### PR DESCRIPTION
**Pull request by albertinisg**
This PR is intended to fix #49. 2 more PR are coming (1 for each branch: `develop`, `master` and `release` branch).
It includes the following changes:
- Use relative links instead of absolute at README.md
- Gather all the documentation under doc/readthedocs folder to make it easier to maintain

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
I think we should have only a pull request to develop and then doing the uplift to the past versions, with git cherry-pick. i would suggest to close not needed PRs

thanks

**Comment by albertinisg** [comment]
PR's #71 and #72 closed and changes will be added in the other branches using cherry-pick

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This application is a smart, context-aware application which allows to manage la
 
 ## Architecture
 
-![Architecture diagram](https://github.com/Fiware/tutorials.TourGuide-App/blob/master/doc/img/archDiagram.png?raw=true "Architecture diagram")
+![Architecture diagram](doc/img/archDiagram.png?raw=true "Architecture diagram")
 
 It includes (for the moment) the following components:
```
Comment (jmcanterafonseca) [inline README.md:44]: Currently, it integrates the following components:

File: doc/readthedocs/architecture/architecture.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Architecture
+
+![Architecture diagram](https://github.com/Fiware/tutorials.TourGuide-App/blob/master/doc/img/archDiagram.png?raw=true "Architecture diagram")
+
+It includes (for the moment) the following components:
+
```
Comment (jmcanterafonseca) [inline doc/readthedocs/architecture/architecture.md:6]: Currently, it integrates ...

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+site_name: FIWARE Tour Guide Application
```
Comment (jmcanterafonseca) [inline mkdocs.yml:1]: Better to name it 'Tour Guide App' as there is a tiny space for displaying it

**Comment by jmcanterafonseca** [comment]
I cannot ack this PR as it needs much work in terms of organizing documentation and on the other hand some of the documentation is obsolete, ex. some NGSIv2 snippets refer to old versions of the payload

With regards to organization of the documentation I would suggest
- Introduction  (pointer to the initial introduction document we developed initially (updated)
- How To
  - Run . here we will detail how to run the complete application and its individual components. 
  - Manage Context Information
  - Configure Authentication and Authorization
  - Manage historical information (Cygnus)
  - Use CEP for event processing
  - Manage sensors with IoT Agents
  - Simulate readings from sensors 
- Deep Dive
  - API
  - Tests
  - Generate a data image
  - Use Context Providers

**Comment by albertinisg** [comment]
@jmcanterafonseca I will start working on this again and, I'm wondering if the purposed arrangement of the documentation could still work or maybe it needs an update (as the latest comment its from April).

**Comment by jmcanterafonseca** [comment]
please, keep this task on hold. We are thinking on a new approach for the
Tour Guide Application documentation and structure, so it does not make
sense to resume this.

2016-06-06 15:14 GMT+02:00 Alberto Martín notifications@github.com:

**Comment by taliaga** [comment]
Hi @jmcanterafonseca @albertinisg,
Any update on this issue and the new documentation structure plans? I'd be able and glad to help with this.

**Comment by jmcanterafonseca** [comment]
@taliaga we are creating a new version of the documentation right now

a first pull request is coming in the next days. We will happy if you can contribute by reviewing or by adding more content once it is landed a first version.

By the way the work is being done at https://github.com/fiware/docs.TourGuide repo ... develop branch

**Comment by jmcanterafonseca** [comment]
closing this. work will be landed on https://github.com/fiware/docs.TourGuide

**Pull request closed without merging**